tests/cypress/integration/case_10_polygon_shape_track_label_points.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Actions on polygon', () => {
+
+    const caseId = '10'
+    const labelName = `Case ${caseId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`
+    const image = `${imageFileName}.png`
+    const newLabelName = `New ${labelName}`
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    const createPolygonShape = {
+        reDraw: false,
+        type: 'Shape',
+        switchLabel: false,
+        pointsMap: [
+            {x: 200, y: 200},
+            {x: 250, y: 200},
+            {x: 250, y: 250},
+            ],
+        complete: true,
+        numberOfPoints: null
+    }
+    const createPolygonTrack = {
+        reDraw: false,
+        type: 'Track',
+        switchLabel: false,
+        pointsMap: [
+            {x: 300, y: 200},
+            {x: 350, y: 200},
+            {x: 350, y: 350},
+            ],
+        complete: true,
+        numberOfPoints: null
+    }
+    const createPolygonShapePoints = {
+        reDraw: false,
+        type: 'Shape',
+        switchLabel: false,
+        pointsMap: [
+            {x: 400, y: 200},
+            {x: 450, y: 200},
+            {x: 450, y: 250},
+            {x: 400, y: 350},
+            {x: 380, y: 330},
+            ],
+        numberOfPoints: 5
+    }
+    const createPolygonTrackPoints = {
+        reDraw: false,
+        type: 'Track',
+        switchLabel: false,
+        pointsMap: [
+            {x: 500, y: 200},
+            {x: 550, y: 200},
+            {x: 550, y: 250},
+            {x: 500, y: 350},
+            {x: 480, y: 330},
+            ],
+        numberOfPoints: 5
+    }
+    const createPolygonShapeSwitchLabel = {
+        reDraw: false,
+        type: 'Shape',
+        switchLabel: true,
+        labelName: newLabelName,
+        pointsMap: [
+            {x: 600, y: 200},
+            {x: 650, y: 200},
+            {x: 650, y: 250},
+            ],
+        complete: true,
+        numberOfPoints: null
+    }
+    const createPolygonTrackSwitchLabel = {
+        reDraw: false,
+        type: 'Track',
+        switchLabel: true,
+        labelName: newLabelName,
+        pointsMap: [
+            {x: 700, y: 200},
+            {x: 750, y: 200},
+            {x: 750, y: 250},
+            ],
+        complete: true,
+        numberOfPoints: null
+    }
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        cy.imageGenerator('cypress/fixtures', image, width, height, color, posX, posY, labelName)
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, image)
```

Comment:
I would suggest do not create a different annotation task for each test in the future (if test doesn't require it). It would be much more efficient if we created a couple of annotation tasks (video, images) with several labels and attributes and used it for all such tests. We might consider such optimization like a separated request. What do you think?

Reply:
Yes, this can be considered as part of a separate task. I will need to think about how best to combine the tests. And of course it will speed up the tests.